tests/ha/run.sh

```diff
@@ -44,10 +46,6 @@ function run() {
 	check_rpc_alive $cur/../bin/check_master_online 127.0.0.1:$MASTER_PORT3
 	check_metric $MASTER_PORT3 'start_leader_counter' 3 -1 1 # master3 is not leader
 
-	# check worker state
-	check_metric $MASTER_PORT1 'dm_master_worker_state{worker="worker1"}' 3 1 3
-	check_metric $MASTER_PORT1 'dm_master_worker_state{worker="worker2"}' 3 1 3
```

Comment:
delete this check beause don''t know who is leader but `dm_master_worker_state` metric only in leader's metrics

Reply:
check_metric $MASTER_PORT1 'dm_master_worker_state{worker="worker1"}' 3 1 3 || check_metric $MASTER_PORT2 'dm_master_worker_state{worker="worker1"}' 3 1 3